File: examples/python_quickstart/test_hello.py
import unittest

from hello import greet


class TestHello(unittest.TestCase):
    def test_greet_default(self) -> None:
        self.assertEqual(greet(), "Hello, world!")

    def test_greet_name(self) -> None:
        self.assertEqual(greet("ai-orch"), "Hello, ai-orch!")


if __name__ == "__main__":
    unittest.main()

File: examples/node_quickstart/test_hello.js
const { test } = require("node:test");
const assert = require("node:assert");

const { greet } = require("./hello");

test("greet default", () => {
  assert.strictEqual(greet(), "Hello, world!");
});

test("greet name", () => {
  assert.strictEqual(greet("ai-orch"), "Hello, ai-orch!");
});
